src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 import UIKit
 
 class DynamicTableViewHtmlCell: UITableViewCell {
-	let textView = HtmlTextView()
+	let webView = HTMLView()
```

Comment:
Will it work with dynamic tableview cell sizing?

Reply:
I tested the Onboarding which worked fine. Do you have something else in mind?

Reply:
Many Subviews inside App Information I guess

Reply:
Fixing the "Nutzungsbedingungen" and "Datenschutz" was the goal of this ticket and all cases (including onboarding) are shown in the video.